db/upgrade-X.X.X-X.Y.Z.sql

```diff
@@ -1,3 +1,33 @@
 --
 -- PacketFence SQL schema upgrade from X.X.X to X.Y.Z
 --
+
+--
+-- Add a primary key to the radacct_log
+--
+
+ALTER TABLE `radacct_log` ADD COLUMN `radacct_log_id` int NOT NULL PRIMARY KEY AUTO_INCREMENT FIRST;
+
+--
+-- Add a primary key to the iplog_history
+--
+
+ALTER TABLE `iplog_history` ADD COLUMN `iplog_history_id` int NOT NULL PRIMARY KEY AUTO_INCREMENT FIRST;
+
+--
+-- Add a primary key to the iplog_archive
+--
+
+ALTER TABLE `iplog_archive` ADD COLUMN `iplog_archive_id` int NOT NULL PRIMARY KEY AUTO_INCREMENT FIRST;
+
+--
+-- Add a primary key to the locationlog
+--
+
+ALTER TABLE `locationlog` ADD COLUMN `locationlog_id` int NOT NULL PRIMARY KEY AUTO_INCREMENT FIRST;
```

Comment:
In your testing, did this add a value to the pre-existing rows ?

If it doesn't, how will the ORM behave ?